packages-next/admin-ui/src/pages/ListPage.tsx

```diff
@@ -14,6 +15,90 @@ type ListPageProps = {
 };
 type Filter = { field: string; type: string; value: JSONValue };
 
+const ListPageHeader = ({ listKey }: { listKey: string }) => {
+  const list = useList(listKey);
+  return (
+    <Stack
+      across
+      marginY="large"
+      gap="medium"
+      css={{
```

Comment:
This makes me think the `Stack` component is incomplete or not the right tool for the job here